helix-front/src/app/cluster/cluster-list/cluster-list.component.ts

```diff
@@ -29,7 +28,6 @@ export class ClusterListComponent implements OnInit {
 
   ngOnInit() {
     this.loadClusters();
-    this.clusterService.can().subscribe((data) => (this.can = data));
```

Comment:
@somecodemonkey I'm honestly not sure what this method is supposed to do. It is not defined in https://github.com/apache/helix/blob/master/helix-front/src/app/cluster/shared/cluster.service.ts 

🤔